codalab/worker/worker.py

```diff
@@ -146,7 +146,10 @@ def create_or_get_network(name, internal, verbose):
         # unused docker networks, or network creation might fail. We only prune docker networks
         # older than 1h, to avoid interfering with any newly-created (but still unused) networks
         # that might have been created by other workers.
-        self.docker.networks.prune(filters={"until": "1h"})
+        try:
+            self.docker.networks.prune(filters={"until": "1h"})
+        except docker.errors.APIError as e:
+            logger.error("Cannot prune docker networks: %s", str(e))
```

Comment:
Should we warn instead of error here? (since this could be benign)

Reply:
you're totally right, thnkas